airbrakes/mock/mock_imu.py

```diff
@@ -27,29 +32,28 @@ class MockIMU(IMU):
 
     def __init__(
         self,
-        log_file_path: Path | bool,
         real_time_simulation: bool,
-        start_after_log_buffer: bool = False,
+        log_file_path: Path | None = None,
+        start_after_log_buffer: bool = True,
     ):
         """
         Initializes the object that pretends to be an IMU for testing purposes by reading from a
-            log file.
+        log file.
+
+        We don't call the parent constructor as the IMU class has different parameters, so we
```

Comment:
i'm still against that btw, it does break LSP. But ok whatever

Reply:
Lol this one is on you. When I was first implementing the MockIMU, I was going to make an abstract class to just represent an IMU, but you said to just have the mock imu inherit from imu

Reply:
but an abstract class implies that MockIMU is different from IMU and all its methods would be implemented differently. We want to make sure MockIMU is as close as possible to the IMU class though, so abstract classes isn't the way

Reply:
I think the "correct" way to do this would be making a parent class both MockIMU and IMU extend. This parent class would contain as much functionality as it could, and both MockIMU and IMU would only have their own specific constructors and implementations of `_fetch_data_loop`

Reply:
I'm fine with that, maybe call it `BaseIMU`?

Reply:
Yeah something like that.